test/test_gen_backend_stubs.py

```diff
@@ -155,7 +155,7 @@ def test_backend_has_no_autograd_key_but_provides_entries(self):
     def test_backend_autograd_kernel_mismatch_out_functional(self):
         yaml_str = '''\
 backend: XLA
-cpp_namespace: torch_msnpu
+cpp_namespace: torch_xla
```

Comment:
Why not vulkan for these ones?

Reply:
Because `backend: XLA`? I am not sure why this was ever `torch_msnpu`, but changing to `torch_xla` seemed to make the most sense here?